requires-testing.txt

```diff
@@ -1,4 +1,4 @@
-pytest<5.0.0
+pytest
```

Comment:
Oh, I didn't notice when <5 came in with https://github.com/plotly/dash/pull/808 - what's the story here? Why did you add it then, why did the restriction get removed now?

Reply:
There was an issue with pytest at the moment, I was looking for a candidate to trigger a cache key not found, a natural pick for the purpose.